airbyte-integrations/connectors/source-sftp-bulk/source_sftp_bulk/spec.json

```diff
@@ -81,6 +81,30 @@
         "examples": ["/logs/2022"],
         "order": 8
       },
+      "max_depth": {
+        "title": "Recursion depth (Optional)",
+        "description": "Number of levels to search for files in the folder path",
+        "type": "integer",
+        "default": "0",
+        "examples": ["0", "1"],
+        "order": 9
+      },
+	  "column_names": {
+		"title": "Column names (Optional)",
+		"description": "A separable string of column names. If _None_ is passed, the header will be used as column names",
+		"type": "string",
+		"default": "_None_",
+		"examples": ["some_col_A|some_col_B"],
+		"order": 10
+	  },
+	  "column_names_separator": {
+        "title": "Separator used to separate the header (Optional)",
+        "description": "A character used to split the column names listed at column_names",
+        "type": "string",
+        "default": "|",
+        "examples": ["|", "#"],
+        "order": 11
+      },
```

Comment:
@diegoquintanav I think you're exposing some unnecessary parameters here. Make the column name of type array of string and read it whatever the user pass to you or at least say you're expecting a list of string separated by comma `,`. This way you can remove one parameter from the config.

Reply:
I agree, will change it to an array in a following commit